helm/defectdojo/templates/django-deployment.yaml

```diff
@@ -93,8 +93,8 @@ spec:
         - name: http
           protocol: TCP
           containerPort: 8081
-        {{- if .Values.django.uwsgi.enable_ptvsd }}
-        - name: ptvsd
+        {{- if .Values.django.uwsgi.enable_debugpy }}
+        - name: debugpy
           protocol: TCP
           containerPort: 3000
```

Comment:
Needs to be a variable now too, per the changes in the Dockerfile.

Reply:
Added in lines 113 to 116